_data/simple-icons.json

```diff
@@ -12902,6 +12902,11 @@
             "hex": "DB0A40",
             "source": "https://www.redbull.com"
         },
+        {
+            "title": "Red Candle Games",
+            "hex": "C2443D",
```

Comment:
> and the color was taken from their brand asset on [redcandlegames.com/presskit](https://redcandlegames.com/presskit).

Not seeing the same color, I'm getting `#D23735`...

Reply:
Yes, you're right. They may updated the color these days.